js/noms/src/bytes.js

```diff
@@ -75,16 +75,13 @@ export function readUtf8(buff: Uint8Array, start: number, end: number): string {
   return buff.toString('utf8', start, end);
 }
 
-export function encodeUtf8(str: string, buff: Uint8Array, dv: DataView, offset: number): number {
-  const size = Buffer.byteLength(str);
+export function encodeUtf8(str: string, buff: Uint8Array, offset: number): number {
```

Comment:
yup. this is cleaner. that was mostly about saving allocations (I forget how exactly).